src/models/user-played-time-response.model.ts

```diff
@@ -0,0 +1,44 @@
+import type { GameConcept } from "./title-concept.model";
+import type { Media } from "./game-media.model";
+
+export interface GetUserPlayedTimeResponse {
+  /** The title ID of this game 
+      @example CUSA26431_00
+      */
+  titleId: string;
+
+  /** Default name for the title */
+  name: string;
+
+  /** Localized name for the title */
+  localizedName: string;
+
+  /** URL to game's console launch screen, from image.api.playstation.com */
+  imageUrl: string;
+
+  /** URL to localized game's console launch screen. Will default to imageUrl if not unique */
+  localizedImageUrl: string;
+
+  /** Type of game being played. Can be ps5_native_game, ps4_game or none */
+  category: string;
+
+  /** Service where this game was acquired from: EA Access, none(purchased/none_purchased, ps_plus, other, */
+  service: string;
```

Comment:
I am wondering if we can strongly type this as well based on the comment.

Reply:
It would probably cause issues (unless we mass-test to fetch several examples and narrow down the edge cases), the format for this is very erratic. I would assume this is not standardized internally somehow?